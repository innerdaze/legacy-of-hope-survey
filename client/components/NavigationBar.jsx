import React, { Component } from 'react'
import { Link } from 'react-router'
import Menu from 'grommet/components/Menu'
import Anchor from 'grommet/components/Anchor'
import Footer from 'grommet/components/Footer'

export default class NavigationBar extends Component {

  render() {
    return (
      <Footer primary={true} appCentered={true}>
        <Menu
          inline={true}
          direction='row'
          responsive={false}
          fill={true}>
          <Link to="/orders">Orders</Link>
        </Menu>
      </Footer>
    )
  }

}
