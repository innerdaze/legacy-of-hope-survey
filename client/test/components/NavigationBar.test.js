import React from 'react'
import NavigationBar from '../../components/NavigationBar'

let wrapper

describe('Testing on NavigationBar...', () => {
  describe('NavigationBar Layout', () => {
    wrapper = shallow(<NavigationBar />)

    test('Expect NavigationBar to have a parent Footer comp and child Menu comp', () => {
      expect(wrapper.find('Footer')).toHaveLength(1)
      expect(wrapper.find('Footer').find('Menu')).toHaveLength(1)

      expect(wrapper).toMatchSnapshot()
    })
  })
})
