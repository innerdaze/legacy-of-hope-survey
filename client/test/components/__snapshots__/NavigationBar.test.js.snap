// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing on NavigationBar... NavigationBar Layout Expect NavigationBar to have a parent Footer comp and child Menu comp 1`] = `
<Footer
  align="center"
  appCentered={true}
  direction="row"
  primary={true}
  responsive={false}
>
  <Menu
    closeOnClick={true}
    direction="row"
    dropAlign={
      Object {
        "left": "left",
        "top": "top",
      }
    }
    fill={true}
    inline={true}
    pad="none"
    responsive={false}
  >
    <Link
      replace={false}
      to="/orders"
    >
      Orders
    </Link>
  </Menu>
</Footer>
`;
